docs/docs/References/Plugin_Config.md

```diff
@@ -53,13 +53,13 @@ srcDir: src/
 schemaFile: schema.json
 pluginType: DIRECT
 language: PYTHON27
+buildNumber: 0.1.0
 ```
-This is a valid plugin config for the plugin with manualDiscovery set to false:
+This is a valid plugin config for the plugin with `manualDiscovery` set to `false` and an `externalVersion`:
```

Comment:
I don't quite understand the "and an `externalVersion`:" portion. Are you just saying here's an example with the externalVersion set? 

if so maybe just add the `set` word at the end? so like:
```
This is a valid plugin config for the plugin with `manualDiscovery` set to `false` and an `externalVersion` set:
```
?

Reply:
yup, changed